### PR TITLE
Add a license of jReject, cc#5500

### PR DESCRIPTION
**Pull request by clairetsai818**
https://github.com/TurnWheel/jReject/blob/master/README.md#licensing

### CODE REVIEW

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

File: ajax/libs/jReject/package.json [hunk 1]
```diff
@@ -35,5 +35,5 @@
       "images/browser*.gif"
     ]
   },
-  "license": "MIT", "GPL-2.0"
+  "license": {"MIT", "GPL-2.0"}
```
Comment (PeterDaveHello) [inline ajax/libs/jReject/package.json]: See https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/vis/package.json#L32-L35 for the right format.

**Comment by clairetsai818** [comment]
replace "license" with "licenses"

**Comment by PeterDaveHello** [comment]
Take a look at the reference and fix it again.

**Comment by clairetsai818** [comment]
Duplicated

**Pull request closed without merging**